.gitignore

```diff
@@ -1,5 +1,6 @@
+source/pages/saml/samlComplianceToolScenarios.rst
```

Comment:
Probably don't want to ignore a specific file like this

Reply:
removed it 👍